CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 11.7.2
+- [fixed] `DataConnectOperationError` now includes underlying server error messages in the debug description instead of a generic decoding error. [Issue](https://github.com/firebase/firebase-ios-sdk/issues/14945)
```

Comment:
nit: For consistency with the other changelog entries, does this render correctly if you do this?

```suggestion
- [fixed] `DataConnectOperationError` now includes underlying server error messages in the debug description instead of a generic decoding error. ([#14945](https://github.com/firebase/firebase-ios-sdk/issues/14945))
```

Reply:
Trying... wasn't sure since it was reported in the Firebase iOS SDK repo issue section and not in the Data Connect repo.

Reply:
Made the change